dev-docs/RFCs/vNext/layer-shader-customization-rfc.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Layer Shader Customization
```

Comment:
`unified-32-and-64-bit-project-api-rfc.md` would seem more appropriate name for this RFC, it doesn't really address customization.

And I don't think it should, as this RFC is already quite self contained and poised to get a quick rubber stamp. I envision that a customization proposal would be a separate thing, starting with a shader injection RFC in luma.gl.

You may want to take a quick look at these:
* https://github.com/uber/luma.gl/blob/master/dev-docs/RFCs/v6.0/shader-injection-rfc.md
* https://github.com/uber/luma.gl/blob/master/dev-docs/RFCs/v6.0/shadertools-improvements-rfc.md